File: chessSquareColor.py
def getChessSquareColor(i,j):
    if i < 1 or i > 7 or j < 1 or j > 7:
        print("Not a valid chess board square!")
    elif (i % 2 == 0 and j % 2 == 0) or (i % 2 != 0 and j % 2 != 0):
        print("white")
    else:
        print("black")


def main():
    i = int(input("Enter chess board number i: "))
    j = int(input("Enter chess board number j: "))
    getChessSquareColor(i,j)


main()

File: chessSquareColor.c
#include <stdio.h>

void getChessSquareColor(int i, int j);

int main(void)
{
    int i,j;
    printf("Enter chess board number i: ");
    scanf("%i", &i);
    printf("Enter chess board number j: ");
    scanf("%i", &j);
    getChessSquareColor(i,j);
    return 0;
}

void getChessSquareColor(int i, int j)
{
    if (i > 7 || j > 7)
    {
        return;
    }
    else if ((i % 2 == 0 && j % 2 == 0) || (i % 2 != 0 && j % 2 != 0))
    {
        printf("white\n");
    }
    else
    {
        printf("black\n");
    }
}